docs/development.rst

```diff
@@ -528,21 +528,22 @@ patch updates are routinely and regularly released.
 
 This entails:
 
-1) Create a PR against ``master`` with the release notes; presently, the
-   release notes are located in ``docs/status.rst``
-2) Once this has been merged, tag the release on GitHub (on the `releases
+#. Create a PR against ``master`` that has the following changes: 
+
+   #. update the release notes in ``docs/status.rst`` with a description of what is in the release 
+   #. modify ``requirements.txt`` to pin the ga4gh packages to specific versions 
+   #. modify ``constraints.txt`` to point to the master branches of all ga4gh packages
```

Comment:
I think you can either delete the constraints file, or comment out the lines pointing to master. We don't want to insert dependency links during a release.